diff --git a/src/main/java/adubbz/nx/analyzer/IPCAnalyzer.java b/src/main/java/adubbz/nx/analyzer/IPCAnalyzer.java
index b2f9f10..e85d539 100644
--- a/src/main/java/adubbz/nx/analyzer/IPCAnalyzer.java
+++ b/src/main/java/adubbz/nx/analyzer/IPCAnalyzer.java
@@ -51,8 +51,8 @@ import org.apache.commons.lang3.tuple.Pair;
 import org.apache.commons.lang3.tuple.Triple;
 import org.python.google.common.collect.Lists;
 import org.python.google.common.collect.Multimap;
-import org.python.google.common.collect.HashBiMap;
-import org.python.google.common.collect.Sets;
+import com.google.common.collect.HashBiMap;
+import com.google.common.collect.Sets;
 
 public class IPCAnalyzer extends AbstractAnalyzer {
diff --git a/src/main/java/adubbz/nx/loader/nxo/NXOAdapter.java b/src/main/java/adubbz/nx/loader/nxo/NXOAdapter.java
index 2bf17df..14b7d9f 100644
--- a/src/main/java/adubbz/nx/loader/nxo/NXOAdapter.java
+++ b/src/main/java/adubbz/nx/loader/nxo/NXOAdapter.java
@@ -118,18 +118,10 @@ public abstract class NXOAdapter extends BinaryReader {
             return this.elfProvider;
         
         long baseAddress = program.getImageBase().getOffset();
-        long memoryProviderLength = 0x0;
-        
-        try 
-        {
-            memoryProviderLength = this.getMemoryProvider().length();
-        } 
-        catch (IOException e) 
-        {
-            Msg.error(this, "Failed to get memory provider length", e);
-        }
+        long memoryProviderLength = this.getMemoryProvider().length();
 
         this.elfProvider = new ElfCompatibilityProvider(program, new LegacyByteProviderWrapper(this.getMemoryProvider(), -baseAddress, memoryProviderLength), this.isAarch32());
         
         return this.elfProvider;
     }
 }
