diff --git a/src/main/java/adubbz/nx/analyzer/IPCAnalyzer.java b/src/main/java/adubbz/nx/analyzer/IPCAnalyzer.java
index b4a275c..22f7115 100644
--- a/src/main/java/adubbz/nx/analyzer/IPCAnalyzer.java
+++ b/src/main/java/adubbz/nx/analyzer/IPCAnalyzer.java
@@ -51,8 +51,8 @@ import ghidra.util.exception.DuplicateNameException;
 import ghidra.util.exception.InvalidInputException;
 import ghidra.util.task.TaskMonitor;
 import org.apache.commons.compress.utils.Lists;
-import org.python.google.common.collect.HashBiMap;
-import org.python.google.common.collect.Sets;
+import com.google.common.collect.HashBiMap;
+import com.google.common.collect.Sets;
 
 import java.io.IOException;
 import java.io.File;
diff --git a/src/main/java/adubbz/nx/loader/nxo/NXOAdapter.java b/src/main/java/adubbz/nx/loader/nxo/NXOAdapter.java
index 2f17104..af5ab64 100644
--- a/src/main/java/adubbz/nx/loader/nxo/NXOAdapter.java
+++ b/src/main/java/adubbz/nx/loader/nxo/NXOAdapter.java
@@ -121,16 +121,7 @@ public abstract class NXOAdapter
             return this.elfProvider;
         
         long baseAddress = program.getImageBase().getOffset();
-        long memoryProviderLength = 0x0;
-        
-        try 
-        {
-            memoryProviderLength = this.getMemoryProvider().length();
-        } 
-        catch (IOException e) 
-        {
-            Msg.error(this, "Failed to get memory provider length", e);
-        }
+        long memoryProviderLength = this.getMemoryProvider().length();
 
         this.elfProvider = new ElfCompatibilityProvider(program, new LegacyByteProviderWrapper(this.getMemoryProvider(), -baseAddress, memoryProviderLength), this.isAarch32());
         
